### PR TITLE
refactor: Move testutils to general location

### PR DESCRIPTION
**Pull request by TerryHowe**
**What this PR does / why we need it**:

The successor PR is big and still lacks coverage without more changes. This PR is for a small part of that change, just moving the console testutils to internal/testutils so we can have one testutils that is ignored by coverage.

### CODE REVIEW

File: internal/testutils/testutils.go
Comment (qweeah) [inline internal/testutils/testutils.go:1]: How about renaming this file to `console.go`?

File: Makefile [hunk 1]
```diff
@@ -33,7 +33,8 @@ LDFLAGS += -X $(PROJECT_PKG)/internal/version.GitTreeState=${GIT_DIRTY}
 
 .PHONY: test
 test: tidy vendor check-encoding  ## tidy and run tests
-	$(GO_EXE) test -race -v -coverprofile=coverage.txt -covermode=atomic -coverpkg=./... ./...
+	$(GO_EXE) test -race -v -coverprofile=coverage-all.txt -covermode=atomic -coverpkg=./... ./...
```
Comment (qweeah) [inline Makefile:36]: How about modifying `-coverpkg`? Like

> $(GO_EXE) test -race -v -coverprofile=coverage-all.txt -covermode=atomic -coverpkg=$(go list ./... | grep -v internal/testutils | tr '\n' ',') ./...

**Review by qweeah — CHANGES_REQUESTED** [review]

File: internal/testutils/testutils.go
Comment (TerryHowe) [reply internal/testutils/testutils.go:1]: Done

File: Makefile (hunk 1 above)
Comment (TerryHowe) [reply Makefile]: Done.

File: .gitignore [hunk 2]
```diff
@@ -36,6 +36,7 @@ debug
 
 # Custom
 coverage.txt
+coverage-all.txt
```
Comment (qweeah) [inline .gitignore:39]: No need to ignore this file

**Review by qweeah — APPROVED** [review]
LGTM with minor suggestion.

File: .gitignore (hunk 2 above)
Comment (TerryHowe) [reply .gitignore]: Done

**Pull request merged**